physics/euler_solver.hpp

```diff
@@ -70,6 +71,18 @@ class EulerSolver {
       Bivector<AngularMomentum, PrincipalAxesFrame> const& angular_momentum,
       Instant const& time) const;
 
+  // Equivalent to this->AttitudeAt(this->AngularMomentumAt(time), time), where
+  // the angular momentum is not needed.
+  AttitudeRotation AttitudeAt(Instant const& time) const;
+
+  // The motion of the body at the given time.  The centre of gravity of the
+  // body moves according to |linear_motion|.
+  RigidMotion<PrincipalAxesFrame, InertialFrame> MotionAt(
+      Instant const& time,
+      DegreesOfFreedom<InertialFrame> const& linear_motion) const;
+
+  R3Element<MomentOfInertia> const& moments_of_inertia() const;
```

Comment:
Reorder: we generally put these small selectors immediately after the constructor.